scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -746,7 +762,23 @@ class FossId internal constructor(
             "${pendingFiles.size} pending files have been returned for scan '$scanCode'."
         }
 
-        return RawResults(identifiedFiles, markedAsIdentifiedFiles, listIgnoredFiles, pendingFiles)
+        val snippets = runBlocking(Dispatchers.IO) {
+            pendingFiles.map {
+                logger.info { "Listing snippet for $it..." }
```

Comment:
This will log immediately instead of when the coroutine starts. If this is not intended, move this to inside `async`.